src/FuzzySearch.js

```diff
@@ -0,0 +1,65 @@
+import Levenshtein from 'liblevenshtein';
+import _at from 'lodash/at';
+import _flatMap from 'lodash/flatMap';
+import _sortBy from 'lodash/sortBy';
+import _sortedUniq from 'lodash/sortedUniq';
+import _keys from 'lodash/keys';
+
+class KeyMap {
+    keys: Array<String>;
+    objectMap: {[String]: Array<Object>};
+    priorityMap: {[String]: number}
+}
+
+const DEFAULT_RESULT_COUNT = 10;
+const DEFAULT_DISTANCE = 5;
+
+export default class FuzzySearch {
+
+    static valuesToKeyMap(objects: Array<Object>, keys: Array<String>): KeyMap {
+        const keyMap = new KeyMap();
+        const map = {};
```

Comment:
These could also use some comments, ('map' is not the most informative variable name)